AK/URL.h

```diff
@@ -83,7 +83,9 @@ class URL {
     ErrorOr<String> serialized_host() const;
     DeprecatedString basename() const;
     DeprecatedString query() const;
+    // NOTE: fragment() is percent-decoded, raw_fragment() is not.
```

Comment:
Got around to getting rid of that in: https://github.com/SerenityOS/serenity/pull/20510